README.md

```diff
@@ -110,6 +114,7 @@ jobs:
           input_path: 'config/acme-openapi.yml'
           config_path: 'config/acme.stainless.yml'
           output_path: 'config/acme-openapi.documented.json'
+          project_name: 'my-stainless-project'
```

Comment:
I think in the context of this README `acme` would make more sense as a value. And we can add a comment or write some explanation if we feel that's not clear enough.